x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -14,6 +14,7 @@ import {
   getPresets,
 } from '@kbn/shared-ux-chrome-navigation';
```

Comment:
There are some links missing after the changes:

- Services
- Infrastructure
- Traces
- Logs
- Machine learning sub links

### Before
<img width="251" alt="image" src="https://github.com/elastic/kibana/assets/5831975/e4bc0913-87c9-4e38-9e7d-5e79539426f8">

<img width="241" alt="image" src="https://github.com/elastic/kibana/assets/5831975/539c7d5f-6c5f-460d-892f-f15fa14f0a20">

### After
![screencapture-localhost-5601-yft-app-observability-alerts-2023-06-12-21_24_35](https://github.com/elastic/kibana/assets/5831975/81f23b05-6186-4e7b-8774-208e560cfee7)